exonum-java-binding/core/rust/src/utils/services.rs

```diff
@@ -53,45 +63,74 @@ mod tests {
     use tempfile::{Builder, TempPath};
 
     #[test]
-    fn no_config() {
-        let res = load_enabled_services("nonexistent");
-        assert!(res.is_err());
+    fn missed_enabled_services_section() {
+        let cfg = "[user_services]\nservice_name1 = '/path/to/artifact1'\nservice_name2 = '/path/to/artifact2'".to_owned();
```

Comment:
Maybe use constants with raw string literal syntax? Would be much more readable.

Reply:
Thanks, somehow I totally forgot about them